connman/src/main.c

```diff
@@ -815,6 +858,17 @@ bool connman_setting_get_bool(const char *key)
 	return false;
 }
 
+unsigned int connman_setting_get_uint(const char *key)
+{
+	if (g_str_equal(key, CONF_ONLINE_CHECK_INITIAL_INTERVAL))
+		return connman_settings.online_check_initial_interval;
+
+	if (g_str_equal(key, CONF_ONLINE_CHECK_MAX_INTERVAL))
+		return connman_settings.online_check_max_interval;
+
```

Comment:
Now silently returns 0 for unknown keys. Would it make sense to have some diagnostic message here?

Reply:
I think there is a pattern for every `connman_setting_get_*` to just silently ignore unknown keys. None of those report anything. And besides, this comes from a upstream change directly. Lets not break the pattern. Each of the values anyways has a default value so I think the caller should then determine based on that, and make debug for invalid return or something.